src/loc.ml

```diff
@@ -85,6 +103,15 @@ let print ppf loc =
           Format.fprintf pp "%s\n%*s\n" line
             stop_c
             (String.make len '^')
+        else if num_lines <= 10 && not (start = loc_file.start) then
+          let lines = file_lines path ~start:start.pos_lnum ~stop:stop.pos_lnum in
+          let last_lnum = Option.map ~f:(fun (lnum, _) -> lnum) 
+                            (List.last lines) in
+          let padding_width = Option.value ~default:0 
```

Comment:
The `None` case will not happen here, so you can use `Option.value_exn` instead of specifying a default.